src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -31,7 +33,15 @@ import { truncateMiddle } from "../../lib/truncate-middle";
 import { keyBy, uniq } from "lodash";
 import { useRefreshAssociatedBalances } from "../../hooks/use-refresh-associated-balances";
 import { useWeb3 } from "../../contexts/web3-context/web3-context";
-import { Colors } from "../../config";
+import { ADDRESSES, Colors, Flags } from "../../config";
+import { BulletHeader } from "../bullet-header";
+import { Routes } from "../../routes/router-config";
+import { Link } from "react-router-dom";
+import vegaWhite from "../../images/vega_white.png";
+import vegaBlack from "../../images/vega_black.png";
+import noIcon from "../../images/token-no-icon.png";
+import { addDecimal } from "../../lib/decimals";
+import { AccountType } from "../../__generated__/globalTypes";
 
 const DELEGATIONS_QUERY = gql`
```

Comment:
probably makes sense to rename the const and query name to something else as its not just delegations anymore

Reply:
maybe:
```suggestion
const VEGA_WALLET_QUERY = gql`
  query VegaWallet($partyId: ID!) {}
```